src/main/java/org/broadinstitute/hellbender/tools/variantdb/arrays/ArraySampleTsvCreator.java

```diff
@@ -0,0 +1,98 @@
+package org.broadinstitute.hellbender.tools.variantdb.arrays;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.variantdb.IngestConstants;
+import org.broadinstitute.hellbender.utils.tsv.SimpleXSVWriter;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class ArraySampleTsvCreator {
+
+    private SimpleXSVWriter sampleMetadataWriter = null;
+    private Map<String, String> metricsMap;
+
+
+    public ArraySampleTsvCreator(String metricsFilepath) {
+        BufferedReader reader = null;
+        try {
+            String columns = null;
+            String values = null;
+            reader = new BufferedReader(new FileReader(metricsFilepath));
+            String line = reader.readLine();
+            while (line != null) {
+                if (!line.startsWith("#") && !line.trim().isEmpty()) {
+                    if (columns == null) {
+                        columns = line;
+                    } else {
+                        values = line;
```

Comment:
Is there only one line in the metrics file? If so, can you put in a check for that?